src/components/goodsBlock/goodsBlock.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import 'bulma/css/bulma.css';
+import classNames from 'classnames';
+import { SortType } from '../types/sortType';
+import { GoodsList } from '../goodsList';
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+type Props = {
+  goods: string[];
+};
+
+export class GoodsBlock extends React.Component<Props, ReorderOptions> {
+  state = {
+    sortType: SortType.NONE,
+    isReversed: false,
+  };
+
+  handleSortByLengthButtonClick = () => {
+    this.setState({
+      sortType: SortType.LENGTH,
+    });
+  };
+
+  handleSortByAlphabetButtonClick = () => {
+    this.setState({
+      sortType: SortType.ALPHABET,
+    });
+  };
+
+  handleReverseButtonClick = () => {
+    this.setState((state) => {
+      return {
```

Comment:
`return` is redundant here, you can directly return the object from the arrow function